core/src/main/java/com/flipkart/gjex/core/config/GrpcConfig.java

```diff
@@ -22,11 +22,22 @@
     @JsonProperty("server.port")
     private int port;
 
+    @JsonProperty("server.max_message_size")
```

Comment:
in the config file we are using either dot separator or camelCase.

Reply:
Ok I will convert this to camelCase